app/controllers/reviews_controller.rb

```diff
@@ -10,6 +22,35 @@ def create
     end
   end
 
+  def update
+    @vinyl = Vinyl.find(params[:vinyl_id])
+    @review = Review.find(params[:id])
+    if @review.update_attributes(review_params)
+      flash[:notice] = "Review was updated."
```

Comment:
single quotes, only use double quotes in ur app if u need to, like for stirng interpolation or if u have the need for a quote apostrophe inside the quotes